src/main/java/org/cubeville/cvgames/models/Game.java

```diff
@@ -21,4 +24,17 @@ public void processPlayerMap(Map<Integer, List<Player>> playerTeamMap) {
 
     protected abstract PlayerState getState(Player p);
 
+    public List<ItemStack> getPlayerCompassContents() {
+        List<ItemStack> items = new ArrayList<>();
+        state.keySet().stream().sorted(Comparator.comparingInt(o -> -1 * getState(o).getSortingValue())).forEach(p -> {
+            ItemStack item = new ItemStack(Material.PLAYER_HEAD);
+            SkullMeta meta = (SkullMeta) item.getItemMeta();
+            meta.setDisplayName("§f" + p.getDisplayName());
+            meta.setOwningPlayer(p);
+            item.setItemMeta(meta);
+            items.add(item);
+        });
+        return items;
+    }
+
```

Comment:
Is there a reason you're defining this as an abstract method that is implemented the same in `Game` and `TeamSelectorGame` instead of just defining the method directly in `BaseGame`?